uber/site_sections/jobs.py

```diff
@@ -36,7 +36,8 @@ def update_counts(job, counts):
 
 @all_renderable(c.PEOPLE)
 class Root:
-    def index(self, session, location=None, message=''):
+
+    def index(self, session, location=None, message='', time=None):
```

Comment:
After you update a shift, the POST returns a redirect to the start time of the shift that was just update. This way, the user doesn't need to scroll from some where else in the calendar. The following code updates the calendar's initial position at load.
''                var curDate = $.urlParam('time');
                if(curDate != null) {
                        $("#shift_cal").fullCalendar( 'gotoDate',curDate);

                } else {
                        $("#shift_cal").fullCalendar('gotoDate', eventList[0].start);
                }
''